### ISSUE TITLE
Group conflicts

### ISSUE THREAD

**Issue by StellaCh** [inspirehep/record-editor#377]
Example: If I have 300 conflicts for authors and I conflict for pub info, it's easy to get lost and make a mistake when approving changes.

Solution: Group conflicts by type so that it's easier to approve/reject them

**Comment by harunurhan** [comment]
to make it clear, we will group them by field right ? because `type` is `add | delete | replace` for a conflict. @StellaCh 

So all of the below will be in the same group:

- `/authors/1/affiliations/value` (replace)
- `/authors/-` (add)
- `/authors/-` (add)
- `/authors/3/full_name` (replace)
- `/authors/5` (delete)

**Comment by StellaCh** [comment]
Yes that sounds good. I was thinking something similar to nested fields where you see `authors`, but then you can still see `full name` or `value` if you extend/collapse the fields

**Comment by harunurhan** [comment]
are `accept/reject all` actions really useful in this case?

**Comment by StellaCh** [comment]
yes still needed. "If there're a lot of authors added, I want to accept them all without going one by one"

**Comment by harunurhan** [comment]
sounds dangerous to me, but alright

**Comment by StellaCh** [comment]
The usual use case is that a paper has a collaboration, but not authors, so the 300 author names are added later. In that case you want to accept them all without clicking one by one. In case we have different actions done in the author names, then curators should be able to expand the actions to make sure they are all the same to avoid human errors

**Comment by harunurhan** [comment]
in that case it makes sense to group by exact path/field, if it is likely that they need to `accept all` only additions.

Which would be like below.

`/authors/1/affiliations/value` (replace)

`/authors/-` (add)
`/authors/-` (add)

`/authors/3/full_name` (replace)

**Comment by StellaCh** [comment]
Yes that looks good
